rest/data/admin.go

```diff
@@ -172,6 +172,22 @@ func (ac *DBAdminConnector) RestartFailedTasks(queue amboy.Queue, opts model.Res
 	}, nil
 }
 
+func (ac *DBAdminConnector) RestartFailedCommitQueueVersions(opts model.RestartTaskOptions) (int, error) {
```

Comment:
Might not use RestartTaskOptions since this isn't restarting tasks. But might still add dry run functionality.